scipy/signal/wavelets.py

```diff
@@ -307,14 +307,95 @@ def ricker(points, a):
     return total
 
 
-def cwt(data, wavelet, widths):
+def morlet2(M, s, w=5):
+    """
+    Complex Morlet wavelet, designed to work with `cwt`.
+
+    Returns the complete version of morlet wavelet, normalised
+    according to `s`::
+
+        exp(1j*w*x/s) * exp(-0.5*(x/s)**2) * pi**(-0.25) * sqrt(1/s)
+
+    .. versionadded:: 1.4.0
```

Comment:
Usually this goes in Notes (I think)

Reply:
> 
> 
> Just a few minor remaining nitpicks.
> 
> For the release notes update, I think we need to say that `cwt` now:
> 
>     1. does time reversal, which is a backward incompatible bugfix for any time-asymmetric wavelets, and
> 
>     2. supports complex wavelets
> 
> 
> Does that sound right?

I've added this to the release notes and sorted out the other comments related to documentation.

Reply:
Didn't get this one

Reply:
This should be sorted now.